src/main/resources/net/uaznia/lukanus/hudson/plugins/gitparameter/GitParameterDefinition/help-tagFilter.html

```diff
@@ -1,6 +1,7 @@
 <div>
     This parameter is used to get tag from git.<br/>
     If is blank, parameter is set to "*".<br/>
+    Regex patterns must be prefixed with with a forward slash (ie /.*).<br/>
```

Comment:
Remove duplicated word
```suggestion
    Regex patterns must be prefixed with a forward slash (ie /.*).<br/>
```